docs/products/metrics.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Aiven for Metrics®
+early: true
+
+---
+
+Aiven for Metrics, powered by Thanos, provides scalable, long-term storage solutions
+for Prometheus metrics. Designed to integrate with Prometheus seamlessly, it extends
+capabilities with efficient historical data retention, high availability, and
+cross-cluster querying. Available for deployment in preferred cloud environments,
+Aiven for Metrics ensures that monitoring and observability stacks can scale with
+business needs, offering comprehensive insights into application performance and
+system health over time.
```

Comment:
This should be on line, at least the first sentence, since all of it will exceed the normal lenght for the meta desc.

Reply:
Fixed